modules/item.rb

```diff
@@ -32,8 +33,8 @@ def author=(author)
   private
 
   def can_be_archived?
-    year = Time.now.year - @publish_date.year
-
-    year > 10
+    current_year = Time.new.year
+    publish_year = Date.strptime(@publish_date, '%Y-%m-%d').strftime('%Y').to_i
+    current_year - publish_year > 10
```

Comment:
Here there is no need to use `Date.strptime(@publish_date, '%Y-%m-%d').strftime('%Y').to_i` since we already made use of the `Date` like so:
```
year = Time.now.year - @publish_date.year
year > 10
```